app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -64,4 +66,192 @@ class SmokeTest {
+
+                // PRIVATE BROWSER
+
+                togglePrivateBrowsingModeOnOff()
+                navigationToolbar {
+                }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+                }.openThreeDotMenu {
+                    verifyThreeDotMainMenuItems()
+                }.openHistory {
+                    verifyTestPageUrl(defaultWebPage.url)
+                }.goBackToBrowser {
+                }.openThreeDotMenu {
+                }.openBookmarks {
+                    verifyBookmarksMenuView()
+                    // verifyEmptyBookmarksList()
+                }.goBackToBrowser {
+                }.openThreeDotMenu {
+                }.openSettings {
+                    verifySettingsView()
+                }.goBackToBrowser {
+                }.openThreeDotMenu {
+                }.openFindInPage {
+                    verifyFindInPageSearchBarItems()
+                }.closeFindInPage {
+                }.openThreeDotMenu {
+                }.addToFirefoxHome {
+                    verifySnackBarText("Added to top sites!")
+                }.openTabDrawer {
+                }.openHomeScreen {
+                    // verifyExistingTopSitesTabs(defaultWebPage.title)
+                }.openTabDrawer {
+                }.openTab(defaultWebPage.title) {
+                }.openThreeDotMenu {
+                }.openAddToHomeScreen {
+                    verifyShortcutNameField(defaultWebPage.title)
+                    clickAddShortcutButton()
+                    clickAddAutomaticallyButton()
+                    verifyShortcutIcon()
+                }.openHomeScreenShortcut(defaultWebPage.title) {
+                }.openThreeDotMenu {
+                }.bookmarkPage {
+                    verifySnackBarText("Bookmark saved!")
+                }.openThreeDotMenu {
+                }.openEditBookmarks {
+                    clickdeleteBookmarkButton()
+                }.confirmBookmarkFolderDeletionAndGoBackToBrowser {
+                    verifySnackBarText("Deleted localhost")
+                }.openThreeDotMenu {
+                }.sharePage {
+                    // verifyShareAppsLayout()
+                    // verifyShareOverlay()
+                }.closeShareDialogReturnToPage {
+                }.openThreeDotMenu {
+                }.refreshPage {
+                    // verifyPageContent(defaultWebPage.content)
+                }.openTabDrawer {
+                    closeTabViaXButton(defaultWebPage.title)
+                }.openHomeScreen {
+                }
+            }
+        }
+    }
+
+    @Ignore
+    @Test
+    fun verifyPageMainMenuItemsListLandscapeMode() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+        activityTestRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE)
+
+        homeScreen {
+            mDevice.waitForIdle(waitingTimeShort)
+            togglePrivateBrowsingModeOnOff()
+            navigationToolbar {

```

Comment:
I think you could start the test from here and so simplify a little bit more the test